src/vm/ceeload.cpp

```diff
@@ -9099,8 +9099,8 @@ void Module::Arrange(DataImage *image)
         //
         // Place hot type structues in the order specifiled by TypeProfilingData array
         //
-        CORBBTPROF_TOKEN_INFO * pTypeProfilingData = profileData->GetTokenFlagsData(TypeProfilingData);
-        DWORD                   cTypeProfilingData = profileData->GetTokenFlagsCount(TypeProfilingData);
+        CORBBTPROF_TOKEN_INFO * pTypeProfilingData = profileData ? profileData->GetTokenFlagsData(TypeProfilingData) : NULL;
```

Comment:
`profileData` was checked for non-null a few lines above. No need to check again.